nautobot/docs/overview/index.md

```diff
@@ -0,0 +1,160 @@
+
+    ---
+    Dive into [how to use Nautobot](../user-guide/feature-guides/getting-started/index.md) and the key components of the core web interface.
+    Learn how [Nautobot Apps](../apps/index.md) can expand Nautobot's functionality.
+
+    ---
+
+- :material-api:{ .lg .middle } **Nautobot APIs!**
+
+    ---
+    Dive into the [REST](../user-guide/platform-functionality/rest-api/overview.md) and [GraphQL](../user-guide/platform-functionality/graphql.md) APIs.
+
+- :material-language-python:{ .lg .middle } **Nautobot SDKs**
+
+    ---
+    Nautobot has a [Python SDK](https://pynautobot.readthedocs.io/) and [Ansible modules](https://galaxy.ansible.com/ui/repo/published/networktocode/nautobot/docs/) to interact with Nautobot in a programmatic way.
+
+</div>
+<!-- markdownlint-enable no-inline-html -->
+
+## Nautobot Apps
+
+Nautobot has a thriving ecosystem of **Apps**, developed as separate projects, for which you can find links to documentation under the [Nautobot Apps](../apps/index.md) section.
+
+<!-- markdownlint-disable no-inline-html -->
+<div class="grid cards" markdown>
+
+- ![Golden Config](../assets/app-icons/icon-GoldenConfiguration.png){style="height: 35px; margin-bottom: 0em" .middle } **Golden Configuration**
+
+    ---
+    [Golden Configuration](https://github.com/nautobot/nautobot-app-golden-config) backs up configurations, generates intended state configurations, compares them for compliance and remediates device configurations.
+
+- ![Device Lifecycle](../assets/app-icons/icon-DeviceLifecycle.png){style="height: 35px; margin-bottom: 0em" .middle } **Device Lifecycle**
+
+    ---
+    [Device Lifecycle](https://github.com/nautobot/nautobot-app-device-lifecycle-mgmt) adds additional capabilities around managing the **hardware** and **software** lifecycle, including the tracking of related **contracts** .
+
+- ![Firewall Models](../assets/app-icons/icon-FirewallModels.png){style="height: 35px; margin-bottom: 0em" .middle } **Firewall Models**
+
+    ---
+    [Firewall Models](https://github.com/nautobot/nautobot-app-firewall-models) helps to model out firewall rules and related objects, including extended ACLs.
+
+- ![SSoT](../assets/app-icons/icon-SSoT.png){style="height: 35px; margin-bottom: 0em" .middle } **SSoT**
+
+    ---
+    [Single Source of Truth](https://github.com/nautobot/nautobot-app-ssot) is the framework to synchronize data from other systems into and out of Nautobot.
+
+- ![ChatOps](../assets/app-icons/icon-ChatOps.png){style="height: 35px; margin-bottom: 0em" .middle } **ChatOps**
+
+    ---
+    [ChatOps](https://github.com/nautobot/nautobot-app-chatops) supports a variety of chat applications, allowing peer teams to conveniently interact with Nautobot and get information about the network.
+
+- ![Circuit Maintenance](../assets/app-icons/icon-nautobot-circuit-maintenance.png){style="height: 35px; margin-bottom: 0em" .middle } **Circuit Maintenance**
```

Comment:
Looks like for self-consistency this should be named `icon-CircuitMaintenance.png`